### DISCUSSION TITLE
What is IBXC

### DISCUSSION THREAD

**Discussion by muco3327** [IBAX-io/go-ibax#1742] [Bounty Hunter]
There is IBXC at this address. The coin given to us? Or something else ?

https://weaver.ibax.io/browse

![image](https://user-images.githubusercontent.com/98721802/151796546-76ba90c3-238d-4967-8c0e-0fdc60000248.png)

**Comment by muco333** [comment]
here

![resim](https://user-images.githubusercontent.com/98265479/151798246-a3dfcb1a-c634-405b-84f8-175bdeb403e1.png)
